src/styles/variables.scss

```diff
@@ -0,0 +1,12 @@
+$gap-space: 1px;
+$weekday:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
+$day-width: 100px;
+$background-gray: #eee;
+$background-pink: pink;
```

Comment:
Avoid using color names in color variables

Consider using background, cell-background, background-hover, etc